aws/cluster.tf

```diff
@@ -1,9 +1,11 @@
 resource "random_uuid" "cluster" {}
 
+resource "time_static" "timestamp" {}
```

Comment:
This time_static resources will capture the timestamp when it is first created, but won't change after that until the resource is destroyed, which I believe is the desired behavior (and the recommended way to handle timestamps in terraform).